### ISSUE TITLE
[TDS] SIF Assessments module Data & Usability Enhancements

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#19404]
Stakeholders were happy with the prototype overall but they had some concerns about making the staff process a bit more streamlined and simplified. This SIF Worksheet Assessment module for staff uses the same data, objects, and many of the same views but it is to be considered a contrasting process that differs from the way a customer would complete a SIF Worksheet Submittal. For this reason, below is a list of intended changes:

- [x] Rename Parallel Reviews to just Reviews (most of the time there will be just one but they need the flexibility to add more)
- [x] Remove Workgroup field
- [x] Email Link to Attachments similar to how a Customer accesses their documents once the process is complete
- [x] Confirmation Email to Creator & Reviewer upon submission
- [x] Review Complete Email to Creator
- [x] Auto Complete the Review instead of Review Decision
- [x] Auto Create the initial Review
- [x] Separate tables for Staff intake for Customer intake
- [x] Separate Customer edit form (since they are required fields but not needed for Staff submittal)
- [x] Set Review Area based on Council District
- [x] Display Rule for ETJ to manually set Review Area
- [x] New Case Due Date field
- [x] Require SIF Service Area & Case Due Date
- [x] New PR Type field with options: Regular (AMANDA) & E-Review (PDox) with help text
- [x] Option attachments on page 2 of staff submittal
- [x] Assign cases based on Review Area
- [x] Hide Customer data in Details when blank for Staff submittal

**Comment by ChrispinP** [comment]
- [ ] Cleaned up bugs
- [ ] Approve Button logic for Case if 100% of reviews complete (auto-submits)
- [ ] Assessment Completion date captured
- [ ] Disabled breadcrumbs & back links where necessary
- [ ] Different details views based on if the form was submitted by staff or customer
- [ ] Pass Review Area to the child Review and set Reviewers automatically
- [ ] Email Signatures now using updated design
- [ ] Spoke to Matiur about the ID abbreviation, they want to go with SWF for SIF Worksheet Final for staff forms. Customer submittals will stay as SIF
- [ ] Tested all forms, everything seems to be working as expected

This module is close to completion and is ready for Final Testing, will inform TDS next week.

**Comment by ChrispinP** [comment]
Informed Matiur that the module is available for testing again. Pending stakeholder feedback & acceptance.

**Comment by TracyLinder** [comment]
**Matiur's Update:** Not all testers have finished their testing. Matiur will followup in the next week or so.

**Comment by TracyLinder** [comment]
Matiur says this is good to close. Thanks @ChrispinP for this work!
